learn/appchains/index.md

```diff
@@ -1,9 +1,20 @@
 ---
 title: Appchain
 description: Discover the features in EVM and Substrate appchains deployed through Tanssi, leveraging user-friendly and easy-to-deploy templates for enhanced capabilities.
-template: subsection-index-page.html
-hide: 
- - toc
- - feedback
+icon: octicons-workflow-24
+template: index-page.html
 ---
 
+# Appchains In Tanssi
```

Comment:
```suggestion
# Tanssi-Powered Networks
```

Reply:
Please check the rest of the guide including the description and swap appchain > networks